src/main/java/com/cloudogu/smeagol/wiki/infrastructure/ScmGit.java

```diff
@@ -0,0 +1,25 @@
+package com.cloudogu.smeagol.wiki.infrastructure;
+
+import com.cloudogu.smeagol.wiki.domain.*;
+import org.eclipse.jgit.lib.PersonIdent;
+import org.eclipse.jgit.revwalk.RevCommit;
+
+import java.time.Instant;
+
+/**
+ * Util class for the ScmGit repositories.
+ */
+public final class ScmGit {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS1118)

Reply:
done